### ISSUE TITLE
how to set empty cell style

### ISSUE THREAD

**Issue by swust-xiaoj** [FancyGrid/FancyGrid#33]
![image](https://user-images.githubusercontent.com/9254851/31886735-30aafd26-b828-11e7-8149-1a0085a9b964.png)
like above picture,I want to set some text when the cell is empty, for instance , set '-' if that

**Comment by MikhailTatsky** [comment]
Do you need that cell display '-' if empty?
If yes than use property `render`
https://fancygrid.com/api/config/defaults/render

```
columns: [{
  render: function(o){
   if(o.value === 'your value'){
     o.value = '-';
   }
   return o;
  }
...
}]
```

If you need to style cell non emty than.
columns: [{
  render: function(o){
   if(o.value === 'your value'){
     o.value = '-';
     o.style['backgroud-color'] = '#6fb270';
   }
   return o;
  }
...
}]
```

**Comment by swust-xiaoj** [comment]
Thanks a lot. 👍 

I tried `console.log(o.value)`, and showed it is an object. So I should traverse the `key` and judge the `value`. 
After that ,the table has a new column, which has text `[object object]`, Maybe I set `width:0` could hidden it.


It works fine.
![image](https://user-images.githubusercontent.com/9254851/31923569-ebc11516-b8ad-11e7-8445-19d3927e9752.png)


But, when I click table head to sort the column, it has a problem.

![image](https://user-images.githubusercontent.com/9254851/31923624-2f2dcc68-b8ae-11e7-95dd-40b439b97630.png)

I have set config `type: 'number'`, May It rendered as `string`.

How can I solve it ?
